### PR TITLE
lesson 12: feat: render section 'homes guest love' through fetch method

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -8,16 +6,18 @@ const createElement =(tagName, className)=>{
     return elem;
 }
 
-const render = (data)=>{
```
Comment (Marinarium) [inline src/main.js]: please use pretify before sending

File: src/main.js [hunk 2]
```diff
@@ -8,16 +6,18 @@ const createElement =(tagName, className)=>{
...
-    data.forEach((item)=>{
+const render = async()=>{
+    const url = 'https://fe-student-api.herokuapp.com/api/hotels/popular';
```
Comment (Marinarium) [inline src/main.js]: I recommend to get data by a separate request
If error??? what about to handle errors
```
function getData(url) {
  //request
 ...
  return data;
}
```

File: src/main.js [hunk 3]
```diff
@@ -28,4 +28,4 @@ const render = (data)=>{
         homesCards.append(homesCard);
     })
 }
-render(dataHomes);
+render();
```
Comment (Marinarium) [inline src/main.js]: image

**Review by Marinarium — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (LudmilaZhuleha) [reply src/main.js]: Corrected

File: src/main.js (hunk 2 above)
Comment (LudmilaZhuleha) [reply src/main.js]: Split for 2 separate functions: render() and getData().

**Pull request merged**